src/components/try-it-out/JsSdkQuickstart.jsx

```diff
@@ -0,0 +1,426 @@
+            </li>
+        </ol>
+        <h2>Verify the app runs</h2>
+        <p>In VS Code execute <code>npm run dev</code> at the terminal prompt. The app should come up at <code>{hostName}</code>. Next you'll create a user to test signin, using the form below.</p>
+ 
+        </div>
+      ) : (
+        <div>
+            <code>
+            <i>To populate the helper content for your tenant, copy your <b>API Token</b> into the form above, then click <b>Configure</b></i>
+            </code>
+            <br/><br/>
+        </div>
+      )}
+        <h2>Get ready to test: create a user and enroll a passkey</h2>
+        <p>In the form below, enter an email address where we can send your passkey invite, then click Create:</p>
+      <div className={classNames(styles["qs-input"], "container")}>
+        <form id="create_id" onSubmit={handleIdSubmit} className={classNames(styles["qs-form"])} autoComplete="off" autoCapitalize="none">
+          <label className={classNames(styles.qslabel)} htmlFor="identityname">Username:</label>
+          <input className={classNames(styles["qs-form-input"])} value={identityName} type="text" id="identityname" name="identityname" onChange={e => setIdentityName(e.target.value)}></input>
+          <label className={classNames(styles.qslabel)} htmlFor="identityemail">Email:</label>
+          <input className={classNames(styles["qs-form-input"])} value={identityEmail} type="text" id="identityemail" name="identityemail" onChange={e => setIdentityEmail(e.target.value)}></input>
+        </form>
+      </div>
+      <div className={classNames(padding["mt-1"])}>
+        <Button
+          name="Create"
+          isLoading={loadingCreate}
+          isDisabled={false}
+          form="create_id">
+        </Button>  { (identityResponse !== null && identityResponse.STATE == "LINK_SENT") ? (<span>You have created a user and passkey successfully! Go ahead and check your email.</span> ) : null}
+      </div>
+      {(createdOneUser) ? (
+        <div id="authenticate-result" className={classNames(padding["mt-1"])}>
+          <p><i>You can update the username and/or email in the form above to create another user and passkey, or otherwise continue with the guidance below.</i></p>
+          <h2>Complete the passkey enrollment</h2>
+          <p>From the browser you wish to use for testing, open the email and click the link. This will initiate a passkey enrollment and registration for your app in Beyond Identity.</p>
+        <h2>Run the app and sign in</h2>
+        <p>If you have not already done so, in VS Code execute <code>npm run dev</code> at the terminal prompt. The app should come up at <code>{hostName}</code>. Here you should be able to click <b>Sign in</b> and sign in to the app as shown below.</p>
+        </div>
+      ) : (
+        <div>
+            <code>
+            <i>Create your test user account and passkey by entering an email address in the form above, then click <b>Create</b></i>
```

Comment:
yes absolutely. Just so I'm clear are you referring to the style of the "Sample App" tip/alert here? https://developer.beyondidentity.com/docs/v1/workflows/sdk-setup
Otherwise please just point me to an example of this style and I'll make it so.